build.gradle.kts

```diff
@@ -10,6 +10,8 @@ buildscript {
     set("targetSdkVersion", 31)
     set("buildToolsVersion", "31.0.0")
   }
+  val kotlinVersion by extra("1.8.0")
```

Comment:
If the root project uses kotlin this should use that version instead, ex my project is using 1.9.22

Reply:
Thanks, yeah, good version

Reply:
kotlin version is now managed from `gradle.properties`, as I've replayed the changes on top of the latest version 👍🏻